skflow/__init__.py

```diff
@@ -117,6 +117,15 @@ def predict(self, X):
             return pred
         return pred.argmax(axis=1)
 
+    def predict_proba(self, X):
+        pred = self._session.run(self._model.predictions,
```

Comment:
I think it would be better to move the _session.run part to def _predict function to have only once and share between predict and predict_proba